### ISSUE TITLE
[BUG] Unable to sign in on https://api-dev.rokwire.illinois.edu/illinois/  Version 7.3.24

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5534]
**Describe the bug**
I was attempting to retest a Web only bug on iOS using the https://api-dev.rokwire.illinois.edu/illinois/ URL. 

I was unable to sign in at any of the locations. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/d7b5fc01-5c4f-4a08-933f-0de14afdb99c)

![Image](https://github.com/user-attachments/assets/76edcfd4-0810-49be-bd03-1d766d6a8041)

![Image](https://github.com/user-attachments/assets/e03af671-ad4c-424b-8357-2785b3f15df7)

**Smartphone (please complete the following information):**
 - Device: iPad Air 
 - OS: 18.6.2
 - Browser Safari
 - App Version 7.3.24

**Additional context**

**Comment by acovher2** [comment]
@vburgett @pmarkhennessy @dobromirdobrev Per comment in #5137 This is a know iOS issue and to be followed under bug #4949.
